### PR TITLE
feat: RBAC

### LINKED ISSUE #4: RBAC
There should be 2 types of USERs
`ADMIN`, and `PARTICIPANT`(default).

- [X] Admin Dashboard

### PR DESCRIPTION
**Pull request by anishshobithps**
- [X] Admin Dashboard
- [X] Data Table
- [X] Per user, data, Payments
- [X] Some cleanup

Some functions are broken, session returns null in the server side, which is weird, or could it be because I changed the session strategy from jwt to database ?

### CODE REVIEW

File: src/app/admin/page.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { auth } from "@/auth";
+import { Payment, User, UserRole } from "@prisma/client";
+import { DataTable } from "./_components/datatable";
+import { columns } from "./_components/columns";
+
+async function getData() {
+  const response = await fetch("http://localhost:3000/api/users");
```
Comment (anishshobithps) [inline src/app/admin/page.tsx]: For some reason, I need to use the absolute URL

Comment (rabilrbl) [reply src/app/admin/page.tsx]: Now way! Can't u use `/api/users` ?

Comment (anishshobithps) [reply src/app/admin/page.tsx]: Resolved in [73ca97d](https://github.com/tiarasjec/website/pull/55/commits/73ca97dc744460017f308ef976e91b5f90f709f4)

**Comment by anishshobithps** [comment]
With the time constraints, I have to merge it, as it is admin part, its not a big deal, if anything else breaks, please make a issue.

**Pull request merged**